libs/@guardian/libs/src/format/ArticleDesign.js

```diff
@@ -0,0 +1,31 @@
+/** @typedef {typeof ArticleDesign[keyof typeof ArticleDesign]} ArticleDesign */
+
+export const ArticleDesign =
+	/** @type {const} @satisfies {Record<string, number>} */ ({
+		Standard: 100,
+		Gallery: 101,
```

Comment:
Why start at `100`?

To prevent clashes with `ArticleTheme` (0-7) and `ArticleDisplay` (200-203)